alma/controllers/hook/GetContentHookController.php

```diff
@@ -92,8 +95,9 @@ public function processConfiguration()
         }
 
         // Down here, we know the provided API keys are correct (at least the one for the chosen API mode)
-        Settings::updateValue('ALMA_LIVE_API_KEY', $liveKey);
-        Settings::updateValue('ALMA_TEST_API_KEY', $testKey);
+        $apiKey = new ApiKeyHelper();
+        $apiKey->setLiveApiKey($liveKey);
+        $apiKey->setTestApiKey($testKey);
```

Comment:
I think you should add the save responsibility to the controller and remove it from the save helper.